server/broadcast.go

```diff
@@ -61,12 +76,18 @@ func (pi *packetInfo) broadcastAll(msgs []*message.Signed) {
 	// If the user has disconnected, then no need to send
 	// the broadcast.
 	if sender == nil {
+		if debugMode {
+			fmt.Printf("[Broadcast] Ignoring from %s because player no longer exists\n", getIP(pi.conn))
+		}
 		return
 	}
 
 	for _, player := range sender.pool.players {
 		// Try to send the message to remaining players even if errors.
-		_ = writeMessage(player.conn, msgs)
+		err := writeMessage(player.conn, msgs)
+		if (err != nil) && debugMode {
+			fmt.Printf("[Broadcast] Continuing to send after write error (%s) to (%s)\n", err, player)
```

Comment:
`fmt.Printf("[Broadcast] Continuing to send after write error  to %s: %s\n", player, err)`

Reply:
Should we just print the IP here rather than the player itself?

Reply:
`fmt.Printf("[Broadcast] Continuing to send after write error to %s: %s\n", getIP(player.conn), err)`

Reply:
I currently have it as printing the standard player debug info as everywhere else (which includes the IP). Even though this is debug output, would you still prefer IP only?

If so, please let me know the logic so I can look everywhere else also.